tracer/build/_build/Build.VariableGenerations.cs

```diff
@@ -31,15 +31,16 @@ Target GenerateVariables
 
             void GenerateConditionVariables()
             {
-                GenerateConditionVariableBasedOnGitChange("isTracerChanged", new[] { "tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation", "tracer/src/Datadog.Trace.ClrProfiler.Native" }, new[] { "tracer/src/Datadog.Trace.ClrProfiler.Native/Debugger" });
-                GenerateConditionVariableBasedOnGitChange("isDebuggerChanged", new[] { "tracer/src/Datadog.Trace.ClrProfiler.Native/Debugger" }, new string[] { });
-                GenerateConditionVariableBasedOnGitChange("isProfilerChanged", new[] { "profiler/src" }, new string[] { });
+                var isNativeLoaderChanged = GenerateConditionVariableBasedOnGitChange("isNativeLoaderChanged", new[] { "tracer/src/Datadog.AutoInstrumentation.NativeLoader" }, new string[] { });
+                GenerateConditionVariableBasedOnGitChange("isTracerChanged", new[] { "tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation", "tracer/src/Datadog.Trace.ClrProfiler.Native" }, new[] { "tracer/src/Datadog.Trace.ClrProfiler.Native/Debugger" }, isNativeLoaderChanged);
+                GenerateConditionVariableBasedOnGitChange("isDebuggerChanged", new[] { "tracer/src/Datadog.Trace.ClrProfiler.Native/Debugger" }, new string[] { }, isNativeLoaderChanged);
+                GenerateConditionVariableBasedOnGitChange("isProfilerChanged", new[] { "profiler/src" }, new string[] { }, isNativeLoaderChanged);
```

Comment:
It sounds like a good idea.
@gleocadie WDYT?

Reply:
Actually, as I said in another comment, I feel we should use the inclusion/exclusion list that we had in the github and consolidated pipeline to consider what is a tracer change and what is a profiler change.

We could even imagine starting to have dedicated pipelines (one for the build, followed by the profiler one, or the tracer one or both). But I think the team actually made the consolidation to one pipeline at some point, so we may need to ask for their feedback first.

Reply:
> As in - move that lest out of those files, and in here, correct? If so, yes, I agree (as they are going to be removed from the yaml files ultimately).

yes that's what I was suggesting

Reply:
Both ideas seem very interesting, however I think it should be addressed as separate PR, since it influences the **whole** CI, and not profiler only, WDYT?

But, adding `profiler/test` to the exclusion list sounds reasonable, will do that shortly.

Reply:
Done in a57b0f2b3a17c1de146e308adc9090025ea38692

Reply:
Forget the 2 pipelines idea. 
Moving the exclusion list that we have in the pipeline for the tracer in here could clearly be done in a later PR.

> But, adding profiler/test to the exclusion list sounds reasonable, will do that shortly.

Looking back to it, if you want to get closer to what was in place for now, you should just include `profiler` here (as both in GH profiler pipelines and in the ultimate pipeline, we filter in/out this whole folder. 
For instance, changing `profiler/cmakelist.txt` should trigger a build I suppose.

Reply:
I don't have a strong opinion, on what path to include.
Done in f497a065270b633f40fec54d242ff3d7e7c90d79.